config/initializers/simple_form.rb

```diff
@@ -92,13 +68,14 @@
   # config.item_wrapper_class = nil
 
   # How the label text should be generated altogether with the required text.
-  # config.label_text = lambda { |label, required| "#{required} #{label}" }
+  config.label_text = lambda { |label, required, explicit_label| "#{} #{label}" }
```

Comment:
so this matches the default just with no indication of whether it is required? probably out of scope but curious if we should add required text to i18n and just use it if it is encouraged
https://github.com/plataformatec/simple_form/blob/master/lib/simple_form.rb#L87